### ISSUE TITLE
ZLib_Ofcep.dll missing when running with debug mode

### ISSUE THREAD

**Issue by KevinLee752** [niessner/BundleFusion#59]
Hi, did anyone know about the error "Cannot find ZLib_Ofcep.dll" when running this code with debug mode? I got this error when running this code on my computer with win10 + VS 2013 + CUDA 10 + RTX 2070, and with debug mode (release mode works fine). I searched the “ZLib_Ofcep.dll” online but got nothing matched. Any advice would be  very appreciated.

**Comment by Lazy-poe** [comment]
I meet the same problem.   Did you solve it？

**Comment by KevinLee752** [comment]
@Lazy-poe No, not solved yet. I also met this problem when running Colmap. So sad....，Did you solve it?

**Comment by zzdengke** [comment]
I met the same problem, how did you finally solve it?

**Comment by E-Devil** [comment]
@KevinLee752 it's 3 years later, i also met this issue when debugging colmap; i solve this issue by copy the missing dll from release version
